tests/foreman/api/test_hostcollection.py

```diff
@@ -19,10 +22,80 @@ def setUpClass(cls):
         ]
 
     @tier1
+    def test_positive_create_with_name(self):
+        """@Test: Create host collections with different names.
+
+        @Feature: Host Collection
+
+        @Assert: The host collection was successfully created and has
+        appropriate name.
+        """
+        for name in valid_data_list():
+            with self.subTest(name):
+                host_collection = entities.HostCollection(
+                    name=name,
+                    organization=self.org,
+                ).create()
+                self.assertEqual(host_collection.name, name)
+
+    @tier1
+    def test_positive_create_with_description(self):
+        """@Test: Create host collections with different descriptions.
+
+        @Feature: Host Collection
+
+        @Assert: The host collection was successfully created and has
+        appropriate description.
+        """
+        for desc in valid_data_list():
+            with self.subTest(desc):
+                host_collection = entities.HostCollection(
+                    description=desc,
+                    organization=self.org,
+                ).create()
+                self.assertEqual(host_collection.description, desc)
+
+    @tier1
+    def test_positive_create_with_limit(self):
+        """@Test: Create host collections with different limits.
+
+        @Feature: Host Collection
+
+        @Assert: The host collection was successfully created and has
+        appropriate limit.
+        """
+        for limit in (1, 3, 5, 10, 20):
+            with self.subTest(limit):
+                host_collection = entities.HostCollection(
+                    max_content_hosts=limit,
+                    organization=self.org,
+                ).create()
+                self.assertEqual(host_collection.max_content_hosts, limit)
+
+    @tier1
+    def test_positive_create_with_unlimited_content_hosts(self):
+        """@Test: Create host collection with different values of 'unlimited
+        content hosts' parameter.
+
+        @Feature: Host Collection
+
+        @Assert: The host collection was successfully created and has
+        appropriate 'unlimited content hosts' parameter value.
+        """
+        for unlimited in (True, False):
+            with self.subTest(unlimited):
+                host_collection = entities.HostCollection(
+                    organization=self.org,
+                    unlimited_content_hosts=unlimited,
+                ).create()
+                self.assertEqual(
+                    host_collection.unlimited_content_hosts, unlimited)
+
+    @tier1
     def test_positive_create_with_system(self):
```

Comment:
I think we should use replace `system` with `content_host`. This applies to all occurrences of system.